plugins/jira-dashboard-backend/src/service/router.ts

```diff
@@ -177,57 +178,99 @@ export async function createRouter(
     },
   );
 
-  router.get(
-    '/avatar/by-entity-ref/:kind/:namespace/:name',
-    async (request, response) => {
-      const { kind, namespace, name } = request.params;
-      const entityRef = stringifyEntityRef({ kind, namespace, name });
-      const { token } = await tokenManager.getToken();
-      const entity = await catalogClient.getEntityByRef(entityRef, { token });
-      const { projectKeyAnnotation } = getAnnotations(config);
+  router.get('/avatar/by-entity-ref/:entityRef', async (request, response) => {
```

Comment:
Please keep the ":kind/:namespace/:name" parameters.

Reply:
updated